### DISCUSSION TITLE
Custom alignment for widget subtitle

### DISCUSSION THREAD

**Discussion by csm10495** [breezy-weather/breezy-weather#1350] [Ideas]
### Describe your idea

![Screenshot_20241005-224727](https://github.com/user-attachments/assets/ee2b49ca-c416-4e61-8cb1-d3f2fb2beadc)

I'm using a custom (long) subtitle. Can we have the option to either center or left align that text?

Thanks.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.8 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
For long texts, I suggest you use Text widget. It can be left or right-aligned

**Reply by HazardousBackup** [reply]
An issue with this is the additional space that gets taken up by the text widget.
I've tried some long subtitles and found some that would/should fit without issue but prematurely wrap instead of allowing for the full width, so even padding to artificially change the alignment likely wouldn't fix their issue.

**Reply by papjul** [reply]
Do you have a screenshot edited to highlight the issue you encounter?

**Reply by HazardousBackup** [reply]
![new](https://github.com/user-attachments/assets/d3030217-49ed-4cf8-a92a-5a508d101ee4)
The text refuses to go past the image of the sun and use the full line length. It appears to be the same as in the original post but their's is set wider so it hasn't wrapped yet but likely would if they added spaces to left align or center it.

**Reply by papjul** [reply]
Ah, you weren't replying to me. In that case, I will suggest the same thing: you can use the Text widget instead.

**Reply by HazardousBackup** [reply]
> Ah, you weren't replying to me. In that case, I will suggest the same thing: you can use the Text widget instead.

No, I was replying to you.
I mentioned that using the Text widget as an addition eats into the screen space.

Though my follow-up was a bit of a non sequitur. What I meant is that since the Text widget eats screen space, it'd be better to use subtitles where possible and the reason this wasn't working for him is because of something that seems generally broken.
